### ISSUE TITLE
Scrobbling is not resumed when the internet goes down and back up

### ISSUE THREAD

**Issue by justalemon** [sttz/TraktRoller#11]
I had this little problem where I was watching "WATATEN!: an Angel Flew Down to Me" and the internet went down. Because of the buffering, I was able to continue watching the episode while the internet was down (a minute or so).

When the internet went up again, I saw that TraktRoller was showing "TypeError: failed to fetch". I ignored it thinking that it would try again after a couple of seconds. Just in case, I checked the progress on Trakt after finishing the season and there was a gap between episodes:

![image](https://user-images.githubusercontent.com/11861253/71350085-3e888580-254f-11ea-904a-6a1c2cfbc8a8.png)

S01E06 was not marked as watched for some reason.

![image](https://user-images.githubusercontent.com/11861253/71350130-5eb84480-254f-11ea-8bda-7be358fba4ea.png)

**Comment by sttz** [comment]
Yeah, there's no retrying or offline tracking. You have to refresh the page to get TraktRoller to try again.

I'm planning some bigger refactorings to allow overriding the episode lookup, which should also make it easier to add retrying in case of connection issues.

In the meantime, the trakt icon and popup show you if an episode has been scrobbled and make it easy to scrobble an episode manually if something went wrong.
